### PR TITLE
#trivial MOBILE-1123 Fixes on Grid field

### PR DESCRIPTION
**Pull request by phcp**

### CODE REVIEW

File: android/library/src/test/java/com/liferay/mobile/screens/ddm/GridFieldTest.kt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import org.junit.experimental.runners.Enclosed
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.util.*
+
+/**
+ * @author Paulo Cruz
+ */
+@RunWith(Enclosed::class)
+class GridFieldTest {
```
Comment (victorlaerte) [inline android/library/src/test/java/com/liferay/mobile/screens/ddm/GridFieldTest.kt]: 🏆🏆🏆🏆🏆🏆

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldSelectView.java [hunk 2]
```diff
@@ -42,6 +42,7 @@
         public void onClick(DialogInterface dialog, int which) {
             clearField();
             alertDialog.dismiss();
+            onClearListener.onClear();
```
Comment (victorlaerte) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldSelectView.java]: I think this may cause NPE

Comment (phcp) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldSelectView.java]: Yeah, that's true. I'll fix it now.

Comment (phcp) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldSelectView.java]: Done!

**Pull request merged**